src/content/en/fundamentals/architecture/building-components/examples/howto-tooltip.md

```diff
@@ -1,10 +1,11 @@
 project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 
-{# wf_updated_on: 2017-07-25 #}
+{# wf_updated_on: 2017-08-14 #}
 {# wf_published_on: 2017-04-06 #}
+{# wf_blink_components: Blink>DOM #}
 
-# HowTo: Components – howto-tooltip {: .page-title }
+# HowTo: Components – <howto-tooltip> {: .page-title }
```

Comment:
You cannot have "markdown" in a heading. You can use `HowTo: Components - &lt;howto-checkbox&gt;`. Encoding the entities will STILL FAIL the test, but I've verified it works and I'll fix the tests asap.